source/uwp/Renderer/dll/CppWinRTIncludes.h

```diff
@@ -34,18 +43,21 @@ namespace winrt
     using namespace ::winrt::Windows::Foundation::Numerics;
     using namespace ::winrt::Windows::Storage::Streams;
     using namespace ::winrt::Windows::System;
-    using namespace ::winrt::Windows::UI;
-	using namespace ::winrt::Windows::UI::Xaml;
-    using namespace ::winrt::Windows::UI::Xaml::Controls;
-    using namespace ::winrt::Windows::UI::Xaml::Controls::Primitives;
-    using namespace ::winrt::Windows::UI::Xaml::Automation;
-	using namespace ::winrt::Windows::UI::Xaml::Automation::Peers;
+    using namespace ::winrt::AdaptiveCards::ObjectModel::Uwp;
+
+    // WinUI 2
+    using namespace ::winrt::Microsoft::UI;
```

Comment:
in general avoid using namespace in header files as it pollutes. Instead create aliases (E.g. `namespace xaml = winrt::Windows::UI::Xaml` etc.)